pkg/pipelinerun/description/description.go

```diff
@@ -60,6 +61,24 @@ STARTED	DURATION	STATUS
 {{ $msg }}
 {{- end }}
 
+
+{{- if .PipelineRun.Spec.Timeouts }}
+
+{{decorate "timeouts" ""}}{{decorate "underline bold" "Timeouts"}}
+{{- $timeout := .PipelineRun.Spec.Timeouts.Pipeline -}}
```

Comment:
```
{{- $timeout := .PipelineRun.Spec.Timeouts.Pipeline -}}
{{- if $timeout }}
 {{decorate "bold" "Pipeline"}}:	{{ $timeout.Duration.String }}
{{- end }
```

Will it not work like

```
{{- if .PipelineRun.Spec.Timeouts.Pipeline}}
 {{decorate "bold" "Pipeline"}}:	{{ $timeout.Duration.String }}
{{- end }
```

Reply:
it's causing some formatting issues(I guess because of `-`) that's why I kept it like that

Reply:
Ohk we can figure out that later or even this is also looking fine.